client/src/api/index.ts

```diff
@@ -18,7 +18,7 @@ const api = {
   getCategories: () => instance.get<Category[]>('/categories'),
 
   createProduct: (product: ProductDto) => instance.post<ProductDetail>('/products', product),
-  getProducts: () => instance.get<Product[]>('/products'),
+  getProducts: (type?: string) => instance.get<Product[]>(`/products${type ? `?type=` + type : ''}`),
```

Comment:
axios에서 쿼리 파라미터 이런 식으로도 넣을 수 있더라구요 괜챃으시면 반영해주세요~!
```suggestion
  getProducts: (type?: string) => instance.get<Product[]>('/products', { params: { type } }),
```

Reply:
오! 이런 방법이 있었네요 반영하겠습니다.